pipelines/openjdk10_nightly_pipeline.groovy

```diff
@@ -0,0 +1,42 @@
+println "building OpenJDK10"
+stage 'build OpenJDK'
+def Platforms = [:]
+Platforms["Linux"] = {
+    node {
+        build job: 'openjdk10_build_x86-64_linux', propagate: false
+    }
+}
+Platforms["Mac"] = {
+    node {
+        build job: 'openjdk10_build_x86-64_macos'
+    }
+}
+Platforms["Windows"] = {
+    node {
+        build job: 'openjdk10_build_x86-64_windows'
+    }
+}
+Platforms["ppc64le"] = {
+    node {
+        build job: 'openjdk10_build_ppc64le_linux'
+    }
+}
+Platforms["aix"] = {
+    node {
+        build job: 'openjdk10_build_ppc64_aix'
+    }
+}
+Platforms["aarch64"] = {
+    node {
+        build job: 'openjdk10_build_arm64_linux'
+    }
+}
+parallel Platforms
+stage 'checksums'
+node {
+    def job = build job: 'openjdk10_build_checksum'
```

Comment:
Is there a reason why the previous builds are launched `build job:` and the last 2 are stored in a variable `def job = build job:`?

Reply:
I couldn't tell you why sorry (don't know the intent of the original author here)